### PR TITLE
Add changelog

### PR DESCRIPTION
**Pull request by yuvipanda**
We're naming releases after Cricket players now, because why not :D cultural diversity, etc!

### CODE REVIEW

**Comment by choldgraf** [comment]
LGTM

also apparently I don't have merge rights :-P

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+* [Carol Willing](https://github.com/willingc)
+* [Jason Kuruzovich](https://github.com/jkuruzovich)
+* [Min RK](https://github.com/minrk/)
+* [Yuvi Panda](https://github.com/yuvipanda/)
+
+<3
+
+### About this cricketer ###
+
+[Wasim Akram](https://en.wikipedia.org/wiki/Wasim_Akram) (وسیم اکرم) is considered by many to be
```
Comment (willingc) [inline CHANGELOG.md]: Love this!

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  what needs to change! 
+
+* A [StorageClass](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#storageclasses)
+  is no longer created by default. This shouldn't affect most new installs,
+  since as of Kubernetes 1.6 most cloud provider installations have a default.
+  If you are using an older version of kubernetes, easiest thing to do is upgrade
+  to a newer version. If not, you can create a StorageClass manually and everything
+  should continue to work.
+  
+* Setting `token.proxy` no longer works - set `proxy.secretToken` instead.
```
Comment (willingc) [inline CHANGELOG.md]: Perhaps:

* `token.proxy` is removed. Use `proxy.secretToken` instead.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  If your `config.yaml` contains something that looks like the following:
+  
+  ```yaml
+  token:
+      proxy: <some-secret>
+  ```
+  
+  you can / should change that to:
```
Comment (willingc) [inline CHANGELOG.md]: ~~can /~~ should

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  
+  ```yaml
+  proxy:
+      secretToken: <some-secret>
+  ```
+
+
+### Features ###
+
+* GitHub Authentication is now supported, thanks to [Jason Kuruzovich](https://github.com/jkuruzovich).
```
Comment (willingc) [inline CHANGELOG.md]: - Added GitHub Authentication support, thanks...

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+* Using an [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) is
```
Comment (willingc) [inline CHANGELOG.md]: * Added Ingress support. If ...

File: CHANGELOG.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+  now supported! If your cluster already has Ingress support (with automatic Let's Encrypt support, perhaps),
+  you can easily use that now.
+* We now add a label to user pods / PVCs with their usernames.
+* Support using a static PVC for user homes or for the hub db. This makes this release usable
```
Comment (willingc) [inline CHANGELOG.md]: user `home` directories or the hub database... where ~~you~~ only ~~have~~ one NFS share is used for the whole hub.

File: CHANGELOG.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+  with clusters where you only have one NFS share that must be used for the whole hub.
+* PostgreSQL is now a supported hub database backend provider.
```
Comment (willingc) [inline CHANGELOG.md]: Hooray!!!!

**Comment by willingc** [comment]
@yuvipanda Jumping on a plane in a few minutes. Feel free to merge as is and we can iterate later.

🏏 <== cricket emoji :smile:

**Comment by willingc** [comment]
@choldgraf Just added you as an admin. You may need to accept the GitHub email.

File: CHANGELOG.md (hunk 1 above)
Comment (yuvipanda) [inline CHANGELOG.md]: Yay for breaking up the cultural hegemony :D

**Pull request merged**